File: fix_keys.py
with open('src/pages/AdminHubPage.tsx', 'r') as f:
    code = f.read()

code = code.replace('<motion.div key="motion-box"', '<motion.div key="feed"', 1)
code = code.replace('<motion.div key="motion-box"', '<motion.div key="filters"', 1)
code = code.replace('<motion.div key="motion-box"', '<motion.div key="client"', 1)

with open('src/pages/AdminHubPage.tsx', 'w') as f:
    f.write(code)

File: fix_keys.js
const fs = require('fs');
let code = fs.readFileSync('src/pages/AdminHubPage.tsx', 'utf8');

code = code.replace(/<motion\.div key="motion-box"/, '<motion.div key="feed"');
code = code.replace(/<motion\.div key="motion-box"/, '<motion.div key="filters"');
code = code.replace(/<motion\.div key="motion-box"/, '<motion.div key="client"');

fs.writeFileSync('src/pages/AdminHubPage.tsx', code);
